src/woo_publications/metadata/api/serializer/category.py

```diff
@@ -0,0 +1,21 @@
+from django.utils.translation import gettext_lazy as _
+
+from rest_framework import serializers
+
+from woo_publications.metadata.models import InformationCategory
+
+
+class InformationCategorySerializer(serializers.ModelSerializer):
+    class Meta:  # type: ignore
+        model = InformationCategory
+        fields = (
+            "identifier",
+            "naam",
+            "naam_meervoud",
+            "definitie",
+            "oorsprong",
+            "order",
+        )
+        extra_kwargs = {
+            "order": {"help_text": _("The order number of the category.")},
```

Comment:
```suggestion
            "order": {"help_text": _("Controls the (default) ordering of categories in result lists.")},
```

I can see that it's an order number from the label and type information already, there's no need to mention that.